hack/tests/subcommand-scorecard.sh

```diff
@@ -12,11 +13,25 @@ set -ex
 
 # the test framework directory has all the manifests needed to run the cluster
 pushd test/test-framework
-commandoutput="$(operator-sdk scorecard --config "$CONFIG_PATH" 2>&1)"
+
+# test to see if v1alpha2 is used from the command line
+commandoutput="$(operator-sdk scorecard --version v1alpha2 --config "$CONFIG_PATH" 2>&1)"
+failCount=`echo $commandoutput | grep -o ": fail" | wc -l`
+if [ $failCount -ne 7 ]
+then
+	echo "expected fail count not equal to output"
```

Comment:
Nit: define `$expectedFailCount` and improve the failure text a bit?
```suggestion
	echo "expected fail count $expectedFailCount, got $failCount"
```

Reply:
agreed, made the change.